### ISSUE TITLE
Stopped working after recent update

### ISSUE THREAD

**Issue by dibsies** [bwp91/homebridge-govee#167]
### What issue do you have? Please be as thorough and explicit as possible.

After updating to the latest version, and updating node to the latest version, the plugin no longer seems to be working. I receive the following message: 

`[23/12/2021, 15:14:34] [Govee] Disabling HTTP client as server is busy at connectionHTTP.login (/usr/local/lib/node_modules/homebridge-govee/lib/connection/http.js:55:15). [23/12/2021, 15:14:34] [Govee] Bluetooth hardware and packages found successfully. [23/12/2021, 15:14:34] [Govee] ***** Disabling plugin. ***** [23/12/2021, 15:14:34] [Govee] ***** No devices found to initialise. *****`

### Details of your setup.
* Do you use (1) Homebridge UI-X (2) Homebridge CLI or (3) HOOBS? 

Homebridge UI-X

* Which version of Homebridge/HOOBS do you have?

Latest

* Which platform do you run Homebridge/HOOBS on (e.g. Raspberry Pi/Windows/HOOBS Box)? Please also mention your version of Node.js/NPM if known.

Pi - Node 16.3.1, NPM 8.1.2

* Which version of this plugin (homebridge-govee) do you have? Has the issue started since upgrading from a previous version?

Latest - started after upgrading this plugin and Node.

* Which Govee devices do you have that are causing issues? Please include product models if applicable.

All Bluetooth strips - none of them are working. I did try changing my password in the app with no luck.

### Please paste any relevant logs below.

```

```

**Comment by charlieschmidt** [comment]
Same.

Strip Lights (H6195) are working fine

Hygro/Thermometers (H5051) are not

```
[24/12/2021, 01:46:11] [Govee] Disabling HTTP client as server is busy at connectionHTTP.login (/usr/lib/node_modules/homebridge-govee/lib/connection/http.js:55:15).
[24/12/2021, 01:46:13] [Govee] Disabling BLE client as no enabled devices configured.
[24/12/2021, 01:46:13] [Govee] [A LED Light] initialising with options {"adaptiveLightingShift":0,"brightnessStep":1,"disableAWS":"unsupported","enableBT":"unsupported","logging":"standard"}.
[24/12/2021, 01:46:13] [Govee] [A LED Light] initialised with id [02:9A:A4:C1:38:9E:13:46].
[24/12/2021, 01:46:13] [Govee] [B LED Light] initialising with options {"adaptiveLightingShift":0,"brightnessStep":1,"disableAWS":"unsupported","enableBT":"unsupported","logging":"standard"}.
[24/12/2021, 01:46:13] [Govee] [B LED Light] initialised with id [D3:58:A4:C1:38:8E:4E:6B].
[24/12/2021, 01:46:13] [Govee] [A LED Light] has been reported [online].
[24/12/2021, 01:46:13] [Govee] [B LED Light] has been reported [online].
[24/12/2021, 01:46:15] [Govee] ✓ Setup complete. Don't forget to ☆ this plugin on GitHub if you're finding it useful!
```

**Comment by xcv58** [comment]
Same here:
```
[12/23/2021, 8:52:03 PM] [Govee] Disabling HTTP client as server is busy at connectionHTTP.login (/usr/lib/node_modules/homebridge-govee/lib/connectio
n/http.js:55:15).
[12/23/2021, 8:52:03 PM] [Govee] Disabling API client as API key not provided.
[12/23/2021, 8:52:04 PM] [Govee] Disabling BLE client as no enabled devices configured.
[12/23/2021, 8:52:04 PM] [Govee] ***** Disabling plugin. *****
[12/23/2021, 8:52:04 PM] [Govee] ***** No devices found to initialise. *****
```

**Comment by dibsies** [comment]
It's starting to sound like it may be an issue on Govee's end

**Comment by xcv58** [comment]
I upgrade to v4.20.1-beta.3 and it works.

**Comment by dibsies** [comment]
I can confirm, upgrading to the beta seems to have worked for me as well. Not sure what changed, but I like it! Lol

**Comment by charlieschmidt** [comment]
Yeah indeed.  Weird.

**Comment by bwp91** [comment]
Just released v4.20.1 which (like beta.3) should fix this issue!

**Comment by dibsies** [comment]
Although the noted update here seemed to resolve these issues - all of my Bluetooth light strips have not been functioning correctly since. After a fresh reboot of homebridge, everything seems to work for roughly 10 minutes. After that, none of them respond to commands from homekit. All devices continue to work through the Govee app. 

Not sure if related, but everything seemed to work flawlessly before the initial update that called for an update of Node from 14.x to 16.x.

The following errors begin showing in my log after ~ten minutes:

`[16/01/2022, 12:02:08] [Govee] [Living Room] could not be updated as BLE took too long at BluetoothLED.updateDevice (/usr/local/lib/node_modules/homebridge-govee/lib/connection/ble.js:165:15).`

@bwp91, please let me know if I should open a new issue or if noting this here does the trick.
